loginserver/database.cpp

```diff
@@ -45,10 +25,10 @@ Database::Database(
 	std::string name
 )
 {
-	this->user = user;
-	this->pass = pass;
-	this->host = host;
-	this->name = name;
+	user = user;
+	pass = pass;
+	host = host;
+	name = name;
```

Comment:
These members need renamed, otherwise it's just local self assignment

Reply:
Alternatively they could be put in the constructor initializer list list and left not-renamed but I think renaming is more clear

Reply:
Good eye; these members don't do anything anyways so I will remove them.

There's a few other places in the loginserver code where member variables are used to pass data around and its bizarre